firmware/defmt-rtt/src/lib.rs

```diff
@@ -206,3 +227,8 @@ unsafe fn handle() -> &'static Channel {
 
     &_SEGGER_RTT.up_channel
 }
+
+fn host_is_connected(channel: &Channel) -> bool {
+    // we assume that a host is connected if we are in blocking-mode. this is what probe-run does.
+    channel.flags.load(Ordering::Relaxed) == BLOCK_IF_FULL
```

Comment:
This breaks with https://github.com/knurling-rs/probe-run/pull/257 - it sets both `BLOCK_IF_FULL` and `NOBLOCK_TRIM`. Instead it should just check if the `BLOCK_IF_FULL` flag is set and ignore the others.

Reply:
Thank you for fixing it in #579 :)